flo-workflow/src/test/java/com/spotify/flo/context/InstrumentedContextTest.java

```diff
@@ -86,19 +86,19 @@ public void callsStatusForFailingTask() throws Exception {
     ));
   }
 
-  Task<String> upstream(int i) {
+  private static Task<String> upstream(int i) {
```

Comment:
Did these test cases just succeed accidentally? If I understood correctly, the primary change in this PR is to enforce a roundtrip of serialisation.

Reply:
Yeah that is what I meant as "accidentally".  When designing the test tasks we didn't always have serializable in mind. Enforcement is good!